packages/sources/elwood-test/src/endpoints/index.ts

```diff
@@ -0,0 +1 @@
+export { cryptoEndpoint } from './crypto'
```

Comment:
Documentation generation is erroring. I think it's expecting this directory to be named `endpoint` instead of `endpoints`.

Reply:
Created a ticket to change this so that it doesn't go through the file structure, but changed it to endpoint here to get it through